addon/lib/Config.jsm

```diff
@@ -80,7 +79,6 @@ const config = {
     "telemetry": {
       "send": true, // assumed false. Actually send pings?
       "removeTestingFlag": false,  // Marks pings as testing, set true for actual release
-      // TODO "onInvalid": "throw"  // invalid packet for schema?  throw||log
```

Comment:
Unclear comment inherited from template. WONTFIX. `Config.jsm` will be getting a facelift when we move to WebExtension Experiments (my next study).